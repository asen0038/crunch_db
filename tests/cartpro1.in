set a 1 2 3
cartprod a
set a
cartprod a
bye
