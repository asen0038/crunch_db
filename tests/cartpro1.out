>
ok

>
[ [1 2 3] ]

>
ok

>
[ [] ]

>
bye
